src/lib/form-field/form-field.md

```diff
@@ -8,6 +8,7 @@ In this document, "form field" refers to the wrapper component `<mat-form-field>
 
 The following Angular Material components are designed to work inside a `<mat-form-field>`:
 * [`<input matInput>` &amp; `<textarea matInput>`](https://material.angular.io/components/input/overview)
+* [`<select matInput>`](https://material.angular.io/components/select/overview)
```

Comment:
Should this be `<select matNativeSelect>`?

(here and elsewhere)

Now that I'm thinking about it, we _could_ deprecate `matInput` and make the selector `matControl` work for both... @mmalerba thoughts?

Reply:
Maybe `matFormControl` for consistency with `mat-form-field`?

Reply:
I'd like it to be something short, `matInput`,`matSelect`, and `matControl` are all fine by me

Reply:
after team discussion, decided to go with matNativeControl